### DISCUSSION TITLE
Web tech

### DISCUSSION THREAD

**Discussion by erf** [zed-industries/zed#6555] [Feature Requests]
Hi, i just wanted to say your web page is very responsive and neat ! I'm curious as to what web technology you've used, like frontend Javascript framework and server technology / language. Are you able to comment on this ? Thanks !

**Comment by iamnbutler** [comment]
👋🏽 Hey thanks!

Here is a non-exhaustive list of the stuff we use on the site:
- Next (12) / Typescript
- *mostly* [Tailwind CSS](https://tailwindcss.com/) for styles
- [Contentlayer](https://www.contentlayer.dev/) for writing in Markdown and turning it into typesafe data
- [Lottie](https://airbnb.design/lottie/) for animations
- Framer motion for animations
- RadixUI for some react primitives
- `next-auth` for interfacing with the Github login / the Zed authentication
- Next has its own API routes built in, so those cover any API needs.
- The chart on the homepage is just vanilla TS

The fonts on the site are [PP Agrandir](https://pangrampangram.com/products/agrandir) and the [iA Writer font family](https://github.com/iaolo/iA-Fonts).

Let me know if there is anything you are curious about in particular.

**Reply by sheikalthaf** [reply]
@iamnbutler hey what editor do you use for website development , because currently tailwind intellisense is not supported in ZED. How you handling that if you are using ZED? I'm also using tailwind in my projects

**Reply by iamnbutler** [reply]
I just write a lot of tailwind and keep the docs site open 😅 We're hopefully going to get tailwind in soon, but I don't want to say the "near future" as I have no clue when exactly.
